integrated_channels/integrated_channel/transmitters/content_metadata.py

```diff
@@ -38,10 +38,11 @@ def transmit(self, payload, **kwargs):
         """
         Transmit content metadata items to the integrated channel.
         """
-        items_to_create, items_to_update, items_to_delete, transmission_map = self._partition_items(payload)
+        items_to_create, items_to_update, items_to_delete, transmission_map, items_catalog_updated_times = \
```

Comment:
Nit: `items_catalog_updated_times` is kind of a misnomer, since this is really a mapping of `content_id` to the time that _content_ last changed.  Maybe something like `content_update_times_by_id`?